io-ballerina/src/io/bytes/block_stream.bal

```diff
@@ -18,7 +18,7 @@ import ballerina/java;
 
 public type Block readonly & byte[];
 
-# BlockStream used to initialize a byte stream.
+# BlockStream used to initialize a byte stream
```

Comment:
I think this should be much more comprehensive unless it's already covered somewhere else like the Module.md. Missing info for example,
- What's a BlockStream?
- What can I do with it?
- When do I need to use a BlockStream?
- Where does it fit in in the IO API architecture?

Also, shouldn't the methods of this class also be documented?

Reply:
Actually, this byte stream, line stream, and CSV stream are for internal usages. However, I added the API docs as you suggested to improve usability. https://github.com/ballerina-platform/module-ballerina-io/pull/37/commits/123095180ab4fe7865b5e69d8623895fe6b978e8